src/renderer/components/dock/terminal/clear-terminal-tab-data.injectable.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import { bind } from "../../../utils";
+import type { TabId } from "../dock/store";
+import type { TerminalStore } from "./store";
+import terminalStoreInjectable from "./store.injectable";
+
+interface Dependencies {
+  terminalStore: TerminalStore;
+}
+
+function clearTerminalTabData({ terminalStore }: Dependencies, tabId: TabId): void {
+  terminalStore.destroy(tabId);
+}
+
+const clearTerminalTabDataInjectable = getInjectable({
+  instantiate: (di) => bind(clearTerminalTabData, null, {
```

Comment:
This bind pattern is just complexity which shouldn't be added. Configuration (e.g. dependencies) shouldn't be mixed with arguments. Arrow function would be much simpler.

Reply:
Inline `clearTerminalTabData`